dataplane/cmd/environment.go

```diff
@@ -120,6 +120,18 @@ func init() {
 					},
 				},
 			},
+			{
+				Name:   "get-request-from-name",
```

Comment:
please do not use `from-name` in the command. We use the flags for qualifying the resource by`name` or `crn`